src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -1087,7 +1087,7 @@ public static PreferenceSettings LoadContent(string content)
             PreferenceSettings settings = null;
 
             if(string.IsNullOrEmpty(content))
-                return new PreferenceSettings();
+                return new PreferenceSettings() { isCreatedFromValidFile = false };
```

Comment:
hmm I guess you chose to not touch constructor since it should be true there?

Reply:
yeah the default constructor should be true as we import the default settings when no file is selected explicitly.